src/Tizen.NET.MaterialComponents/Components/MList.cs

```diff
@@ -0,0 +1,78 @@
+﻿using ElmSharp;
+using System;
+using System.Collections.Generic;
+
+namespace Tizen.NET.MaterialComponents
+{
+    public class MList : GenList, IColorSchemeComponent
+    {
+        HashSet<GenItem> _realizedItems = new HashSet<GenItem>();
+        Color _none = new Color(0, 0, 0, 0);
```

Comment:
What `none` means? What is different with default and `Color.Black`?

Reply:
It is different with `Color.Black`, I changed the name to `_transparent`